src/routes/blog/post/understand-data-queries/+page.markdoc

```diff
@@ -84,8 +84,8 @@ const databases = new Databases(client);
 
 // OR operator example
 const movieData1 = databases.listDocuments(
-    '[DATABASE_ID]',
-    '[COLLECTION_ID]',
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
```

Comment:
I am assuming the refactor for blog post is done in another PR? @adityaoberai

Reply:
Yes, that's a different PR